user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/GQI/Extensions/Ad_hoc_data/Ad_hoc_Metadata.md

```diff
@@ -0,0 +1,22 @@
+---
+uid: Ad_hoc_Metadata
+---
+
+# Linking rows to DataMiner objects
+
+Just like the built-in data sources, you can link the rows of your ad hoc data source to specific DataMiner objects. This will allow you to feed these objects in the Dashboards and Low-Code Apps to other components.
```

Comment:
Was a bit confused by 'Just like the built-in data sources', it sounded like its expected from me (as user) to know how that's done. Might be clearer to first explain that certain built in data sources contain metadata to map the row on a DataMiner object + why you should do that (feeds).